<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border: 2px solid grey;
border-radius: 8px;
background-color: rgb(57, 0, 0);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="No2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>125</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DecButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CosButton">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>210</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>Cos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>55</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>55</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TanButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>Tan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AcosButton">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>250</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>ACos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ZeroButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MultipButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 0);</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="LCD">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(206, 225, 211);
font: 75 14pt &quot;Times New Roman&quot;;
</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="DivButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>125</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 0);</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AsinButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>ASin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Reset">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>55</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SubButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 0);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SinButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>Sin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AtanButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>250</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>ATan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>125</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>55</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 85, 0);</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DzeroButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EqualButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>125</y>
       <width>41</width>
       <height>65</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>125</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="No9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>55</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ExpButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>Exp</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ToNegButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>+/-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RoundButton">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>370</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
</string>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SquaredButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>Sq</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SqRtButton">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>290</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>√</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>291</width>
       <height>5</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>2</y>
       <width>5</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="LogButton">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>330</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LognButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>330</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>Log n</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CbRtButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>80</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 175, 137);
</string>
     </property>
     <property name="text">
      <string>³√</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>5</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>5</width>
       <height>401</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
